demo/JD-recommendation/oneflow_process/train_deepfm.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+DEVICE_NUM_PER_NODE=1
+DEMODIR="$1"
+DATA_DIR="$DEMODIR"/openmldb_process/out
+PERSISTENT_PATH="$DEMODIR"/oneflow_process/persistent
+MODEL_SAVE_DIR="$DEMODIR"/oneflow_process/model_out
+MODEL_SERVING_PATH="$DEMODIR"/oneflow_process/model/embedding/1/model
+
+python3 -m oneflow.distributed.launch \
+    --nproc_per_node $DEVICE_NUM_PER_NODE \
+    --nnodes 1 \
+    --node_rank 0 \
+    --master_addr 127.0.0.1 \
+    deepfm_train_eval_JD.py \
+      --disable_fusedmlp \
+      --data_dir "$DATA_DIR" \
+      --persistent_path "$PERSISTENT_PATH" \
+      --table_size_array "11,42,1105,200,11,1295,1,1,5,3,23,23,7,5042381,3127923,5042381,3649642,28350,105180,7,2,5042381,5,4,4,41,2,2,8,3456,4,5,5042381,10,60,5042381,843,17,1276,101,100" \
+      --store_type 'cached_host_mem' \
+      --cache_memory_budget_mb 1024 \
+      --batch_size 10000 \
```

Comment:
After we change a smaller dataset, the --batch_size parameter should be changed to 1000 to make the test AUC not so bad